test/models/invitation_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class InvitationTest < ActiveSupport::TestCase
+  def setup
+    @event = Event.create(name: "Moe's bar", date: Time.zone.now)
+    3.times { |number| Invitation.create(event: @event, email: "new_user#{ number }@example.com") }
```

Comment:
This is creating 3 invitations before each test. But is just used on "Delete casade" test... I would move there. 
And 3 invitations?  I would iterate less as possible (2 if you want to check multiple invitations for same event).